docs/reference/hipblas-api-functions.rst

```diff
@@ -129,6 +136,14 @@ Graph support (also referred to as stream capture support) for hipBLAS depends o
 If rocBLAS is the backend, see the :doc:`rocBLAS <rocblas:index>` documentation.
 Similarly, if CUDA cuBLAS is the backend, see the cuBLAS documentation.
 
+Custom data types
+=================
+
+hipBlas defines the ``hipblasBfloat16``, ``hipblasComplex``, and
```

Comment:
hipblasComplex and hipblasDoubleComplex should be removed here as we use hipComplex types now.

Reply:
Fixed.